.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "hidapi"]
 	path = hidapi
-	url = https://github.com/PastaJ36/hidapi.git
+	url = https://github.com/libusb/hidapi.git
```

Comment:
Does the libusb fork work with 32 and 64 bit builds? I remember that’s something I changed last time

Reply:
They've got CI builds for 64 & 32 for Windows, so should be alright